src/chrome/extension/scripts/background.ts

```diff
@@ -50,7 +60,16 @@ function initUI() : UI.UserInterface {
   chromeBrowserApi = new ChromeBrowserApi();
   // TODO (lucyhe): Make sure that the "install" event isn't missed if we
   // are adding the listener after the event is fired.
-  chrome.runtime.onInstalled.addListener(chromeBrowserApi.bringUproxyToFront);
+  chrome.runtime.onInstalled.addListener(() => {
+    chrome.tabs.query({currentWindow: true, active: true}, function(tabs){
+        // Do not open the extension when it's installed if the user is
+        // going through the inline install flow.
+        // TODO (lucyhe): update this link. Current URL is for testing only!
+        if (tabs[0].url.indexOf("uproxysite.appspot.com/chrome-install") == -1) {
```

Comment:
Can you add a check for both uproxysite.appspot.com/chrome-install and uproxy.org/chrome-install?